2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,12 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let favoriteRecipe = {
+      title: "Spageti Bolones",
+      serving:2,
+      ingredients:["salt","beef","sauce"]
+};
+  console.log(`${favoriteRecipe.title}\n`,`Serves: ${favoriteRecipe.serving}` ) ;
+  console.log("Ingredients:");
+favoriteRecipe.ingredients.forEach(Element=>console.log(Element));
```

Comment:
great job, I think we wanted you to have 5 receipts and show all of the keys to each receipt in one line (console.log them)